src/compass/rdr2geo.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+'''wrapper for rdr2geo'''
+
+import os
+import time
+
+import isce3
+import journal
+
+from compass.utils.runconfig import RunConfig
+from compass.utils.yaml_argparse import YamlArgparse
+
+def run(cfg):
+    '''run rdr2geo with provided runconfig'''
+    info_channel = journal.info("rdr2geo.run")
+    error_channel = journal.error("rdr2geo.run")
+
+    t_start = time.process_time()
+    info_channel.log("starting rdr2geo")
+
+    # common rdr2geo inits
+    dem_raster = isce3.io.Raster(cfg.dem)
+    epsg = dem_raster.get_epsg()
+    proj = isce3.core.make_projection(epsg)
+    ellipsoid = proj.ellipsoid
+
+    for burst in cfg.bursts:
+        # get isce3 objs from burst
+        rdr_grid = burst.as_isce3_radargrid()
+        isce3_orbit = burst.orbit
+
+        # init output directory in scratch
+        output_path = f'{cfg.scratch_path}/{burst.burst_id}'
```

Comment:
Is there a way to put together the topo layer and the reference burst? For this workflow, this bundle of rasters will be our product